### PR TITLE
On add homepage branch

### PR DESCRIPTION
**Pull request by jojo987N**
### Hello Microverse Project : Homepage 
In this first step we :

- Added html index file
- Added css  file
- Set up and testing linters config

### CODE REVIEW

**Comment by jojo987N** [comment]
I have tried every possible way to resolve the Lighthouse linter but without success. Because according to the logs, it starts a server, while my site is static. So I installed express js, i added lighthouserc js config file,  but the problem did not resolve.

File: README.md [hunk 1]
```diff
@@ -1 +1,200 @@
...
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Linters</summary>
+  <ul>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice way to add the README, However, kindly add the header as shown in the README template. This was included to give your profile a professional view. 👍 

## Template view 
image

## Your view:
image

File: README.md [hunk 2]
```diff
@@ -1 +1,200 @@
...
+sh
+npm install
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly delete the comments from your code, this makes it noisy, and not so professional. As a good programmer in the making, please adhere to this to build proficiency 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,200 @@
...
+  cd hello-microverse
+  git clone git@github.com:jojo987N/hello-microverse.git
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly refer to line `79` regarding deleting the comment 👍

File: README.md [hunk 4]
```diff
@@ -1 +1,200 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ]  Please delete the comments from your code, this was placed to give instructions. As a good programmer in the making, please adhere to this to build proficiency 👍

File: README.md [hunk 5]
```diff
@@ -1 +1,200 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please complete this section to show people how to support your project 👍

File: README.md [hunk 6]
```diff
@@ -1 +1,200 @@
...
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly delete the comment highlighted, and add a usage on how to use the project. e.g, the `index.html` file. This way, you give your viewers a chance to have a step by step process of viewing your code 👍

File: README.md [hunk 7]
```diff
@@ -1 +1,200 @@
...
+  </ul>
+  <ul>
+    <li><a href="https://expressjs.com/">Node Js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Version Control</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Git</a></li>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly delete the section that does not relate to your project, and add those that do. E.g `CSS`, `HTML`
- This way you personalise the codebase 👍

File: README.md [hunk 8]
```diff
@@ -1 +1,200 @@
-# hello-microverse
\ No newline at end of file
+# 📖 # Hello Microverse
+
+**Hello Microverse** is a HTML and CSS project repository with best practices, follow Github flow and a descriptive and easy to understand README file .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice way to add the [README](https://github.com/microverseinc/readme-template) , However, kindly add the header as shown in the README template. This was included to give your profile a professional view. 👍 

## Template view 
image

## Your view:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @jojo987N        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅Proper use of `.gitignore`
✅ Proper use of index files

  
# Required Changes ♻️

- [ ] I must admit you have really put in the effort so far in this project, however, You still need to work on the failing linters. All linters must be passed for your project 
-  [ ] Kindly add more description to your pull request summary. Please follow this [Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to see a detailed view 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 9]
```diff
@@ -1 +1,143 @@
...
+
+  - Setup Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ABDULALI3468) [inline README.md:143]: **[OPTIONAL BUT NECESSARY FOR IMPROVEMENT]** Please consider adding a License file to your repository to make it more professional and in accordance with the project requirement. Currently, as there is no License file this link is resulting in a 404 ERROR.

**Review by ABDULALI3468 — APPROVED** [review]
Hi @jojo987N , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**